source/PlayerInfo.cpp

```diff
@@ -249,7 +249,12 @@ void PlayerInfo::Load(const string &path)
 					reputationChanges.emplace_back(
 						GameData::Governments().Get(grand.Token(0)), grand.Value(1));
 		}
-
+		else if(child.Token(0) == "tribute received")
+		{
+			for(const DataNode &grand : child)
+				if(grand.Size() >= 2)
+					tributeReceived[GameData::Planets().Get(grand.Token(0))] = grand.Value(1);
```

Comment:
Using SetTribute here wouldn't work, as SetTribute calls GameData::Planets().Find() instead of .Get(). We shouldn't .Find() during loading, but we also shouldn't .Get() during during gameplay, hence not using SetTribute here or changing SetTribute to match.